style.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 62%;
+  background-color: #e5e5e5;
+}
+
+.menu-wrap {
+  position: absolute;
+  width: 375px;
+  height: 70px;
+  padding: 20px;
+  background-color: white;
+}
+
+.brand {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.bar {
+  width: 30px;
+  height: 3px;
+  margin: 5px;
+  display: block;
+  background-color: #6070ff;
+}
+
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.wrapper {
+  width: 375px;
+  height: 718px;
+  background-image: url(/images/header.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-color: white;
+  border-bottom-left-radius: 8em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 20px;
+}
```

Comment:
- [ ] I suggest removing the **fixed width** from the wrapper class since it is not really responsive on mobile resolution between **min 375px** && **max 768px**, by following the mobile-first approach you don't really need to set fixed sizes to sections.

![image](https://user-images.githubusercontent.com/68030297/219329864-45ef2d11-40be-4867-ad55-5b60aa87dab2.png)

> **NOTE 🎯:** If you decided removing the width on the wrapper, please ensure to fix the position of it's background image as well 👏🚀